### PR TITLE
Introduce AGL based demo product

### PR DESCRIPTION
**Pull request by iartemenko**
This based on prod-devel commit: 0aa3954ab78576aed08aa68f7c475c1a74d6de4a

### CODE REVIEW

**Comment by iartemenko** [comment]
WIP label removed, ready for review.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb [hunk 1]
```diff
@@ -133,5 +133,5 @@ do_install() {
 FILES_${PN} = " \
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb]: Looks like this commit is not needed anymore.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/display-manager.service [hunk 2]
```diff
@@ -4,7 +4,7 @@ Requires=dbus.service weston.service
 After=dbus.service weston.service
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/display-manager.service]: Also add Requires and After fields as following:

Requires=dbus.service weston-ready.service
After=dbus.service weston-ready.service

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/sndbe.service [hunk 3]
```diff
@@ -12,4 +12,4 @@ ExecStopPost=/usr/bin/xenstore-write drivers/sndbe/status dead
 Restart=always
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/sndbe.service]: This commit is not needed. sndbe.service should be the same as in prod-devel.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/displbe.service [hunk 4]
```diff
@@ -1,11 +1,11 @@
 [Unit]
 Description=Display backend
-Requires=proc-xen.mount weston.service
-After=proc-xen.mount weston.service
+Requires=proc-xen.mount weston-ready.service
+After=proc-xen.mount weston-ready.service multi-user.target
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/displbe.service]: remove multi-user.target from After field.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston-ini-conf/hdmi-a-2-0.cfg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+[output]
+name=HDMI-A-2
+mode=1920x1080
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston-ini-conf/hdmi-a-2-0.cfg]: mode=1920x1080@60

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-multimedia/pulseaudio/files/default.pa [hunk 6]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/pulseaudio -nF
```
Comment (al1img) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-multimedia/pulseaudio/files/default.pa]: This commit is not needed.

**Comment by iartemenko** [comment]
Done with all comments.

**Pull request merged**